### ISSUE TITLE
Package Manager für HiveMQ CE

### ISSUE THREAD

**Issue by dobermai** [hivemq/hivemq-community-edition#67]
HiveMQ CE should be available via Package Manager like Homebrew, deb, RPM, ... to ease the installation process. This should be part of every community release to trigger a job that releases the new HiveMQ Version for all Package Manager.

**Comment by frittentheke** [comment]
@dobermai  while you are "at it" ... please consider providing a Helm3 chart for deployments on Kubernetes. With your nice discovery extension (https://github.com/hivemq/hivemq-dns-cluster-discovery-extension) and the Docker images ready for Kubernetes, this would be an awesome addition to get HiveMQ up an running in such environments as well.

**Comment by fraschbi** [comment]
HI @frittentheke,

Thank you for the input.
This is the HiveMQ Community Edition Repository. Clustering is not included in that version.

Regards,
Florian

**Comment by frittentheke** [comment]
@fraschbi fair point about the clustering being available in the commercial editions only.

Still a Helm chart is what the <s>cool</s> cloud kids require to deploy onto Kubernetes nowadays - Single node or clustered.
And apart from this general point about proving Helm chart, what keeps you from providing mostly the same Helm chart for CE as well as for the other editions? Just add a variable to chose the edition which then switches the container image as well.

To help with "marketing" the potential feature upsell, there could be a comment line in the values.yaml just above the "clustering:" section stating that this requires a license which you could prepare a Secret/ConfigMap template (https://helm.sh/docs/chart_template_guide/accessing_files/#configmap-and-secrets-utility-functions) for.

A common and well maintained Helm Chart would just be a great addition.

**Comment by Remit** [comment]
Closing issue because of inactivity.
